core/os/android/adb/commands.go

```diff
@@ -148,6 +148,31 @@ func (b *binding) StartService(ctx context.Context, a android.ServiceAction, ext
 	return b.Shell("am", args...).Run(ctx)
 }
 
+// ForceStop stops everything associated with the given package.
+func (b *binding) ForceStop(ctx context.Context, pkg string) error {
+	args := append([]string{
+		"force-stop", pkg,
+	})
+	return b.Shell("am", args...).Run(ctx)
+}
+
+// GetSystemProperty returns the system property in string
+func (b *binding) GetSystemProperty(ctx context.Context, name string) (string, error) {
+	var stdout bytes.Buffer
+	if err := b.Shell("getprop", name).Capture(&stdout, nil).Run(ctx); err != nil {
```

Comment:
You can use:

```go
	if res, err := d.Shell("getprop", name).Call(ctx); err == nil {
```

There are plenty of place we do this already. Might be worth moving those calls over to this method.